packages/react/src/types/index.ts

```diff
@@ -77,7 +80,10 @@ export type InfiniteQueryConfig<Data, Error> = Pick<
   | 'onError'
   | 'onSettled'
   | 'onSuccess'
->
+> & {
+  /** Scope the cache to a given context. */
+  contextKey?: string
```

Comment:
Wonder if it's better to call this something else so it doesn't get confused with React Context or React Query's `context` option if we choose to add something similar in the future? Maybe `scopeKey` or `cacheKey`?

Reply:
Yeah, maybe `scopeKey` is a better fit. I have a feeling people may take `cacheKey` too literally (ie. think it is the _actual_ query key).